backend/Dockerfile

```diff
@@ -1,12 +1,16 @@
-FROM alpine:3.11
+FROM alpine:3.12
```

Comment:
This update is totally unrelated to this pull request and should have been at least a different commit if not PR.